### ISSUE TITLE
Error display test fails in 1.9

### ISSUE THREAD

**Issue by agrimm** [quackingduck-archive/exemplor#7]
This runcoderun http://runcoderun.com/agrimm/exemplor/builds/d6221af4298766f1dd2d75b244f65c2a0ae2dade/3/ruby_191 indicates that in ruby 1.9, the expectation for the backtrace isn't met. Maybe the expectation needs to be lowered.

**Comment by quackingduck** [comment]
Yeah. I haven't got 1.9 running on my machine yet. I'm probably going to have more energy for these kind of bugs once I that happens.

**Comment by agrimm** [comment]
Fixed in the current version of my branch, as indicated by runcoderun run http://runcoderun.com/agrimm/exemplor/builds/d7c30d47d8c86eebdfd07d2d31686a9391503a6a/1/ruby_191
